### ISSUE TITLE
Maintability of buildpkg images

### ISSUE THREAD

**Issue by nqb** [Orange-OpenSource/buildpkg-centos#3]
Hello,

I would like to know if `buildpkg` on Docker Hub are still maintained due to [gitlab-buildpkg](https://gitlab.com/Orange-OpenSource/gitlab-buildpkg) project.

My understanding is that `gitlab-buildpkg-tools` doesn't use anymore `buildpkg` Docker hub images but rely on another [container registry](https://gitlab.com/Orange-OpenSource/gitlab-buildpkg/container_registry)

Thanks for your feedback.

**Comment by lehcim** [comment]
> My understanding is that gitlab-buildpkg-tools doesn't use anymore buildpkg Docker hub images but rely on another container registry

Yes, we reworked the image build toolchain with kaniko and the primary registry for buildpkg images is now the gitlab.com registry, because Gitlab pipelines are  really easier to use than Docker Hub pipelines.

Currently the images on Docker Hub are out of sync but we still plan to maintain them, may be with a redirection in the Dockerfile, something like

```
# Dockerfile for DockerHub registry
FROM registry.gitlab.com/orange-opensource/gitlab-buildpkg/centos:7
# end of file
```

**Comment by nqb** [comment]
Thanks for your input @lehcim.

Keep up the good work ;-)
